include/aoapplication.h

```diff
@@ -187,16 +231,33 @@ class AOApplication : public QApplication
   // returns a list of call words
   QStringList get_callwords();
 
-  // returns whatever preanimations should always play or not
-  bool get_always_pre_enabled();
+  // returns the current theme
+  QString get_theme();
```

Comment:
If you were documenting the above functions, you could as well document these 6-7 new functions, too.

Reply:
See #dev